src/__tests__/metadata.ts

```diff
@@ -0,0 +1,92 @@
+import { Server } from 'http'
+import { ethers } from 'ethers'
+import { goerliProvider } from '@/helpers/providers'
+import abiForName from '@/helpers/contracts/abiForName'
+import data from '@/data'
+import env from '@/helpers/env'
+import getBadge from '@/helpers/getBadge'
+import getContract from '@/helpers/getContract'
+import getOriginalContractName from '@/helpers/getOriginalContractName'
+import nodeHtmlToImage from 'node-html-to-image'
+import renderReact from '@/helpers/renderReact'
+import request from 'supertest'
+import runApp from '@/helpers/runApp'
+import shutdown from 'http-graceful-shutdown'
+
+jest.setTimeout(50000)
+
+describe('Metadata endpoint', () => {
+  const goerliDerivative = '0x74168621B4827BAa4532F9a41caf1f3E7B936c36'
+  const externalDerivative = '0xd6B27C7B755B88B58db881A4FfD80B14EC702b49'
+  const emailDerivative = '0x62bC06bafE0ADe5D591f5D50A86a4173614697dD'
+  const tokenId = '0'
+
+  let server: Server
+
+  beforeAll(async () => {
+    server = await runApp()
+  })
+
+  afterAll(async () => {
+    await shutdown(server)
+    return new Promise<void>((resolve, reject) => {
+      server.close((err) => {
+        err ? reject(err) : resolve()
+      })
+    })
+  })
+
+  async function testMetadata(tokenAddress: string) {
+    const response = await request(server).get(
+      `/metadata/${tokenAddress}/${tokenId}`
+    )
+    const contract = new ethers.Contract(
```

Comment:
I think you can import `Contract` without `ethers`

Reply:
sure, ty